force_bdss/mco/i_evaluator.py

```diff
@@ -0,0 +1,35 @@
+import logging
+
+from traits.api import Interface, Instance
+
+from .base_mco_model import BaseMCOModel
+
+log = logging.getLogger(__name__)
+
+
+class IEvaluator(Interface):
+    """Class that is used as a bridge between the BDSS application and
+    MCO runner. It can be passed in as an argument to a MCO run and
+    used to evaluate the state of a system for a given set of
+    parameters, returning a set of KPIs. This avoids the need for the
+    MCO to obtain any information regarding the Envisage application."""
+
+    #: An instance of the MCO model information
+    mco_model = Instance(BaseMCOModel)
```

Comment:
This class is really just a container for a `BaseMCOModel` and any extra non-MCO specific information that could be used to evaluate the KPIs at a given data point. In our `BDSSApplication`, systems are described by `Workflow` objects, so we have also included a `WorkflowEvaluator` class that can evaluate the workflow KPIs.